wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -17,7 +17,7 @@ import org.bitcoins.wallet.models._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
-sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
+abstract class Wallet extends DLCWallet with UnlockedWalletApi {
```

Comment:
@Christewart don't know the Wallet well enough to judge but is this what we want to be doing type-wise?